sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -482,5 +585,46 @@ public class TestPoco
             public string Name { get; set; }
             public string Value { get; set; }
         }
+
+        public class EventHubTestInitialOffsetFromStartEndJobs
+        {
+            public static void ProcessSingleEvent([EventHubTrigger(TestHubName)] string evt,
+                       string partitionKey, DateTime enqueuedTimeUtc, IDictionary<string, object> properties,
+                       IDictionary<string, object> systemProperties)
+            {
+                if (evt == _testId)
+                {
+                    _eventWait.Set();
+                }
+            }
+        }
+
+        public class EventHubTestInitialOffsetFromEnqueuedTimeJobs
+        {
+            private const int ExpectedEventsCount = 2;
+
+            public static void ProcessMultipleEvents([EventHubTrigger(TestHubName)] EventData[] events)
+            {
+                foreach (EventData eventData in events)
+                {
+                    string message = Encoding.UTF8.GetString(eventData.Body.ToArray());
+
+                    // filter for the ID the current test is using
+                    if (message == _testId)
```

Comment:
Do we still need this condition considering we have a hub per test?

Reply:
I think so since you can set an EventHub override environment variable.

Reply:
Removed this.